content/sensu-go/5.16/guides/monitor-sensu.md

```diff
@@ -0,0 +1,163 @@
+---
+title: "Monitor Sensu with Sensu"
+linkTitle: "Monitor Sensu with Sensu"
+description: "Make sure your Sensu components are properly monitored. This guide describes best practices and strategies for monitoring Sensu."
+weight: 230
+version: "5.16"
+product: "Sensu Go"
+platformContent: False
+menu: 
+  sensu-go-5.16:
+    parent: guides
+---
+
+
+- [Monitor your Sensu backend instances](#monitor-your-sensu-backend-instances)
+- [Monitor your Sensu agent instances](#monitor-your-sensu-agent-instances)
+- [Monitor your Sensu API instances](#monitor-your-sensu-api-instances)
+- [Monitor your Sensu dashboard instances](#monitor-your-sensu-dashboard-instances)
+
+This guide describes best practices and strategies for monitoring Sensu with Sensu. 
+It explains how to ensure your Sensu components are properly monitored, including your Sensu backend, agent, API, and dashboard instances.
+
+To completely monitor a Sensu stack (Sensu backend, agent, API, and dashboard), you will need to have at least one other independent Sensu instance.
+A single Sensu stack cannot monitor itself completely because if some components are down, Sensu cannot create events.
+
+This guide requires Sensu plugins.
+For more information about using Sensu plugins, see [Install plugins with assets][16].
+
+_**NOTE**: This guide assumes you are not using Sensu clustering. The strategies in this guide are still useful for monitoring cluster members, but you should follow [Run a Sensu cluster][17] to effectively monitor clustered instances._
+
+## Monitor your Sensu backend instances
+
+The host running the `sensu-backend` should be monitored in two ways:
+
+* Locally by a `sensu-agent` process for operating system checks/metrics and Sensu services that are not part of the Sensu event system (like the dashboard)
+* Remotely from an entirely independent Sensu stack to monitor Sensu components that must be running for Sensu to create events. 
+
```

Comment:
Ok, so the only way that I can think of visualizing this without a wall of text would be a table:

|                                                     | service checks | port checks | prometheus collector check | api queries as checks |
|-----------------------------------------------------|----------------|-------------|----------------------------|-----------------------|
| Singular backend                                    |                |             |                            |                       |
| Clustered backends                                  |                |             |                            |                       |
| Distributed singular backend instances              |                |             |                            |                       |
| Distributed clustered backend instances             |                |             |                            |                       |
| Distributed singular backend instances (federated)  |                |             |                            |                       |
| Distributed clustered backend instances (federated) |                |             |                            |                       |

I'm only one ☕️ in , so this might be a bad way of doing this, but I'll defer to @sensu/reliability for some sanity checking here.

Reply:
I think this is the best way to do a high level over view, then we could provide an example for each column or maybe have a suggested plugin column.

Reply:
I added a draft table, although I used different headings because otherwise the table columns aren't wide enough to accommodate the text. The content in the table cells is just placeholder and should be updated. If I misunderstood the purpose of this table, please let me know and I can fix it!